### PR TITLE
Upgrade amqp verison

### PR DESCRIPTION
**Pull request by mothxs**
I upgraded the amqp version because it is incompatible with the version in `dftask_publisher` package.
https://github.com/doofinder/dftasks_publisher/blob/master/mix.exs#L31

So when you add both of them in doomanager, `mix deps.get` fails with the error:

image

So following [this](https://github.com/pma/amqp/wiki/3.0-Release-Notes#breaking-changes-and-upgrade-guide) upgrade guide I upgraded the dependency.

### CODE REVIEW

**Review by agutierrezrodriguez — COMMENTED** [review]
Did you tried it?

I think that the code needs some changes too.

**Comment by mothxs** [comment]
> Did you tried it?
> 
> I think that the code needs some changes too.

It seems to work when I send it from console:

image

Am I missing something?

File: mix.exs [hunk 1]
```diff
@@ -51,15 +51,15 @@ defmodule EventsManager.MixProject do
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
-      extra_applications: [:lager, :logger],
```
Comment (valeas90) [inline mix.exs:54]: Was `lager` not used ?

Comment (mothxs) [reply mix.exs:54]: Yes but following this upgrade guide https://github.com/pma/amqp/wiki/3.0-Release-Notes#lager-is-no-longer-needed-to-be-in-extra_application it not needed anymore in extra applications.

**Review by valeas90 — APPROVED** [review]

**Pull request merged**